Lib/turtle.py

```diff
@@ -3382,6 +3404,26 @@ def filling(self):
         """
         return isinstance(self._fillpath, list)
 
+    @contextmanager
+    def fill(self):
+        """Create a filled shape
```

Comment:
```suggestion
        """Create a filled shape.
```

I'm really nitpicking (really). But sometimes there are things I see when I look at them again after.